x-pack/plugin/esql/src/main/antlr/EsqlBaseParser.g4

```diff
@@ -22,28 +34,30 @@ query
 sourceCommand
     : explainCommand
     | fromCommand
+    | metaCommand
     | rowCommand
-    | metricsCommand
     | showCommand
-    | metaCommand
+    // in development
+    | {devVersion()}? metricsCommand
```

Comment:
gate metrics command by a semantic predicate.

Reply:
Small problem - some things are gated behind feature flags and those flags can be enabled by setting a jvm parameter *OR* using a snapshot build. I don't remember precisely what is gated, but it's pretty close to this list.

Doing it with a feature flag is a little more convenient that the SNAPSHOT version directly because, on a release, folks can set the jvm parameter and test against that.

Reply:
With the mechanism in place, it's easy to extend this to include feature flags - the main issue is handling this outside the server (e.g. Kibana).